chain-abci/src/app/rewards.rs

```diff
@@ -155,15 +142,20 @@ mod tests {
         let state = app.last_state.as_ref().unwrap();
         let top_level = &state.top_level;
         let reward1 = monetary_expansion(
+            total_staking.into(),
             top_level.rewards_pool.tau,
-            total_staking,
-            top_level.rewards_pool.minted,
-            &top_level.network_params,
+            top_level
+                .network_params
+                .get_rewards_monetary_expansion_r0()
+                .as_millis(),
+            top_level.network_params.get_rewards_reward_period_seconds() as u64,
         );
+        println!("reward1: {}", reward1);
```

Comment:
info! , also more explanation

Reply:
Removed this line since it's only for debugging purposes.